### DISCUSSION TITLE
Sample data set 'games.csv'

### DISCUSSION THREAD

**Discussion by alotia** [dflib/dflib#380] [Q&A] [answered]
I wanted to try out some of the (hockey games) example have shown in your talks*. They use a sample `games.csv` file. I looked arounds a bit, but cannot find the file. If it is available for others to use, can you please provide a pointer to it. Thanks!

* I attended the SFBay Java Meetup one and found the project very interesting.

**Comment by andrus** [comment, marked as answer]
Sure. Here it is. 
[games.csv](https://github.com/user-attachments/files/17639125/games.csv)

**Reply by alotia** [reply]
That was quick! Thanks very much @andrus
